site/src/pages/release-notes/1.12.0.mdx

```diff
@@ -0,0 +1,204 @@
+---
+date: 2021-10-05
+---
+
+## 🌟 New features
+
+- You can now bind a virtual host to a specific port. #3798 #3833
```

Comment:
Could you sort and group items by components? Core changes could come first and other changes could come later in a grouped way, e.g. Spring changes could put together. More popular and important features could come earlier than others, e.g. gRPC and Kotlin changes could come before Spring changes.